docs/src/typescript/README.md

```diff
@@ -170,6 +172,30 @@ const isSunny = currentWeather[0] === 'sunny'
 Enums are a feature added to JavaScript by TypeScript which allows for describing a value which could be one of a set of possible named constants. Unlike most TypeScript features, this is not a type-level addition to JavaScript but something added to the language and runtime. Because of this, it’s a feature which you should know exists, but maybe hold off on using unless you are sure. You can read more about enums in the [Enum reference page](https://www.typescriptlang.org/docs/handbook/enums.html).
 :::
 
+### const, let and var
+
+Typescript allows to define two types of variables:
+
+- mutalbe variables which are declared using the `let` keyword. The legacy `var` keyword can also be used but is not recommended because of its way of handling scope can be misleading.
```

Comment:
petite correction d'anglais:

```suggestion
- Mutable variables which are declared using the `let` keyword. The legacy `var` keyword can also be used but is not recommended because its way of handling scope can be misleading.
```

Reply:
done